package.json

```diff
@@ -69,15 +69,15 @@
     "@babel/plugin-proposal-pipeline-operator": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
-    "@storybook/addon-storysource": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
-    "@storybook/react": "^5.3.19",
-    "@storybook/source-loader": "^5.3.19",
+    "@storybook/addon-storysource": "^6.1.11",
+    "@storybook/addons": "^6.1.11",
+    "@storybook/react": "^6.1.11",
+    "@storybook/source-loader": "^6.1.11",
     "@testing-library/dom": "^6.1.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.1.3",
     "@types/node": "^14.14.22",
-    "@types/react": "^16.9.11",
+    "@types/react": "^17.0.0",
```

Comment:
I wonder whether this is a good approach since we haven't upgrade React to 17.0.0 yet, why is this change needed as part of this PR?

Reply:
Agreed, I switched it back to `"@types/react": "^16.9.11"`